{"name": "Проверка валидности поля Повтор пароля", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.html", "trace": "venv\\lib\\site-packages\\selene\\core\\wait.py:102: in for_\n    return fn(self._entity)\nvenv\\lib\\site-packages\\selene\\core\\wait.py:56: in __call__\n    return self._fn(entity)\nvenv\\lib\\site-packages\\selene\\core\\query.py:47: in <lambda>\n    text: Query[Element, str] = Query('text', lambda element: element().text)\nvenv\\lib\\site-packages\\selene\\core\\entity.py:179: in __call__\n    return self._locator()\nvenv\\lib\\site-packages\\selene\\core\\locator.py:34: in __call__\n    return self._locate()\nvenv\\lib\\site-packages\\selene\\core\\entity.py:1529: in <lambda>\n    f'{self}.element({by})', lambda: self.driver.find_element(*by)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:976: in find_element\n    return self.execute(Command.FIND_ELEMENT, {\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:321: in execute\n    self.error_handler.check_response(response)\nvenv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:242: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\nE     (Session info: chrome=113.0.5672.94)\n\nDuring handling of the above exception, another exception occurred:\ntests\\test_login.py:125: in test_field_repeat_password\n    page.registration_field_repeat_password()\npages\\home.py:271: in registration_field_repeat_password\n    message_error = self.get_element_text(self.message_error_repeat_password_text_var, \"Текст ошибки поля повторить пароль не верный или текст отсутствует\")\npages\\base.py:27: in get_element_text\n    return element.get(query.text)\nvenv\\lib\\site-packages\\selene\\core\\entity.py:116: in get\n    return self.wait.for_(query)\nvenv\\lib\\site-packages\\selene\\core\\wait.py:125: in for_\n    raise self._hook_failure(failure)\nE   selene.core.exceptions.TimeoutException: Message: \nE   \nE   Timed out after 10s, while waiting for:\nE   browser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\nE   \nE   Reason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\nE     (Session info: chrome=113.0.5672.94)\nE   \nE   Screenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.png\nE   PageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.html"}, "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1684321343353, "stop": 1684321343402}, {"name": "Нажать Зарегистрироваться", "status": "passed", "start": 1684321343403, "stop": 1684321343551}, {"name": "Валидация поля Повторить Пароль", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\home.py\", line 271, in registration_field_repeat_password\n    message_error = self.get_element_text(self.message_error_repeat_password_text_var, \"Текст ошибки поля повторить пароль не верный или текст отсутствует\")\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 27, in get_element_text\n    return element.get(query.text)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 116, in get\n    return self.wait.for_(query)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "steps": [{"name": "Заполнение поля '' текстом ''wer23''", "status": "passed", "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616C790>"}, {"name": "text", "value": "'wer23'"}, {"name": "fieldName", "value": "''"}], "start": 1684321343551, "stop": 1684321343637}, {"name": "Взять текст из элемента ''Текст ошибки поля повторить пароль не верный или текст отсутствует''", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 10s, while waiting for:\nbrowser.element(('xpath', \"//div[contains(text(),'недопустимый пароль')]\")).text\n\nReason: NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//div[contains(text(),'недопустимый пароль')]\"}\n  (Session info: chrome=113.0.5672.94)\n\nScreenshot: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.png\nPageSource: file://C:\\Users\\Qasquad1\\.selene\\screenshots\\1684320964491\\1684320964496.html\n\n", "trace": "  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\pages\\base.py\", line 27, in get_element_text\n    return element.get(query.text)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\entity.py\", line 116, in get\n    return self.wait.for_(query)\n  File \"C:\\Users\\Qasquad1\\PycharmProjects\\lime-qa\\venv\\lib\\site-packages\\selene\\core\\wait.py\", line 125, in for_\n    raise self._hook_failure(failure)\n"}, "parameters": [{"name": "element", "value": "<selene.core.entity.Element object at 0x00000122A616EC10>"}, {"name": "allureText", "value": "'Текст ошибки поля повторить пароль не верный или текст отсутствует'"}], "start": 1684321350690, "stop": 1684321361124}], "start": 1684321343551, "stop": 1684321361124}], "attachments": [{"name": "Скринтшот", "source": "656d1647-8ea8-4aab-9d0c-e8b0f086261d-attachment.png", "type": "image/png"}, {"name": "Логи консоли", "source": "8e32023f-b76c-4eba-bc35-2893baf71e1d-attachment.json", "type": "application/json"}, {"name": "log", "source": "6fab9f80-bc57-43fa-95f0-f0f92254d19d-attachment.txt", "type": "text/plain"}], "start": 1684321343353, "stop": 1684321361125, "uuid": "3eaa024e-4047-4f9a-86d6-2baa8ea7df5e", "historyId": "2395a9b2a2c906142f6768d7c5d88907", "testCaseId": "19208eadd6ac9712b22d1ff353e8cf28", "fullName": "tests.test_login.TestLogin#test_field_repeat_password", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "8372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}